internal/commands/root.go

```diff
@@ -265,6 +274,21 @@ func getFilters(cmd *cobra.Command) (map[string]string, error) {
 	return allFilters, nil
 }
 
+func validateExtraFilters(filterKeyVal []string) []string {
+	// Add support for state = exclude-not-exploitable, will replace all values of filter flag state to "TO_VERIFY;PROPOSED_NOT_EXPLOITABLE;CONFIRMED;URGENT"
+	for filterKey, extraFilters := range extraFilter {
```

Comment:
you over twice on the extraFilter instead of run also on the filterKeyVal
why do you ask only on  filterKeyVal[0]
I don't understand this logic..